src/routes/start/[...id]/index.tsx

```diff
@@ -109,10 +116,21 @@ export default component$(() => {
     isDirty.value = false;
     transactions.value = await computeSplit(store.members);
 
+    nav(
+      `/start/?billName=${
+        searchParams.get("billName") || BILL_NAME_DEFAULT
+      }&memberCount=${
+        store.members.length ||
+        searchParams.get("memberCount") ||
+        MEMBER_COUNT_DEFAULT.toString()
+      }`,
+    );
```

Comment:
This navigate isn't needed, if you had some issues here it might be because the database is currently paused (free plan of supabase 😅).
Technically with the DB up after hitting Split for the first time, on line 144 there's already a navigate to `/start/{id}`.

Reply:
Ok, now I understand 🤣
I added this feature to update the memberCount so that when the page was reloaded, the deleted members were deleted.
No worries, I correct without problems